README.md

```diff
@@ -2,13 +2,43 @@
 [![Build Status](https://travis-ci.org/zalando/dress-code.svg?branch=master)](https://travis-ci.org/zalando/dress-code)
 [![Dependency Status](https://david-dm.org/zalando/dress-code.svg)](https://david-dm.org/zalando/dress-code)
 [![devDependency Status](https://david-dm.org/zalando/dress-code/dev-status.svg)](https://david-dm.org/zalando/dress-code#info=devDependencies)
+
 # The Dress Code
 
-The Dress Code is a framework agnostic, atomic design, BEM, style library which is designed for maintainability and modularity.
+The Dress Code is a style library / style guide which is designed for maintainability and modularity.
+
+Used by Zalando Brand Solutions department to solve the challenge of consistency between multiple apps.
+
+We open source the project because we think that can be a good example of how collaboration between UX Designers and Developers can be enhanced with the help of technology.
```

Comment:
Suggestion: 
We open sourced the project because we think it's a good example of how collaboration between UX Designers and Developers is enhancing consistency and user experience with the help of technology.